themes/login-forms/templates/SilverStripe/Forms/CheckboxField_holder.ss

```diff
@@ -5,7 +5,14 @@
         <% if $RightTitle %> $RightTitle<% end_if %>
     </label>
     <% if $getAttribute('title') %>
-        <i tabindex="0" class="font-icon-help-circled" title="$getAttribute('title')" data-toggle="tooltip"></i>
+        <i class="login-form__help-icon font-icon-help-circled"
```

Comment:
Would it make more sense to make this a `<a>` or a `<button>` so we can tag to it directly without having to set a `tabindex` on it and avoid using `cursor: pointer;`

Reply:
Not a biggy.